src/dlgProfilePreferences.cpp

```diff
@@ -162,7 +162,12 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
                                                                      "does not have a needed glyph (a font's individual character/symbol) to represent "
                                                                      "the grapheme (what is to be represented).  Clearing this checkbox will allow "
                                                                      "the best alternative glyph from another font to be used to draw that grapheme.</p>")));
-
+    checkBox_runAllKeyBindings->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
+                                           .arg(tr("<p>Check all Key-bindings against key-presses.</p><p><i>Versions of Mudlet prior to <b>3.9.0</b> would "
```

Comment:
Don't need to load users with history. I'd have the option itself be called:

> Allow duplicate keybindings

Tooltip:

> Allow multiple keys to match on the same combination. With this off, the only the first keybinding that matches will run

Reply:
I think we have to make mention of the past because, I think at least Brenex, has scripts that rely on the *only one key-binding at a time works* and so we have to explain why it is disabled by default to avoid breakage even though that is not what we would have done if we'd thought about it before.

Reply:
We did think about it before, and we made it match once for a reason, and it's good that it stays a default like that - it's the better option... it's why it has worked so well for so long and still does!

Don't need to explain the history behind it, let's keep text to the minimum.